src/components/bottom-widget.js

```diff
@@ -73,51 +156,75 @@ export default function BottomWidgetFactory(TimeWidget, AnimationControl) {
     const enlargedFilterIdx = filters.findIndex(
       f => f.enlarged && f.type === FILTER_TYPES.timeRange
     );
-    const isAnyFilterAnimating = filters.some(f => f.isAnimating);
+    const animatedFilter = filters.find(f => f.isAnimating);
+
     const enlargedFilterWidth = isOpen ? containerW - sidePanelWidth : containerW;
 
     // show playback control if layers contain trip layer & at least one trip layer is visible
-    const animatedLayer = layers.filter(
+    const animatableLayer = layers.filter(
       l => l.config.animation && l.config.animation.enabled && l.config.isVisible
     );
 
-    const readToAnimation = Array.isArray(animationConfig.domain) && animationConfig.currentTime;
+    const readyToAnimation =
+      Array.isArray(animationConfig.domain) && Number.isFinite(animationConfig.currentTime);
     // if animation control is showing, hide time display in time slider
-    const showFloatingTimeDisplay = !animatedLayer.length;
-    const showAnimationControl = animatedLayer.length && readToAnimation;
+    const showFloatingTimeDisplay = !animatableLayer.length;
+    const showAnimationControl = animatableLayer.length && readyToAnimation;
```

Comment:
you can get rid of readyToAnimation since you are using it in here and do the following:
```
const showAnimationControl = useMemo(() => 
  animatableLayer.length && Array.isArray(animationConfig.domain) && Number.isFinite(animationConfig.currentTime), 
 [animatableLayer.length, animationConfig.domain, animationConfig.currentTime]
);
```